app/models/article.rb

```diff
@@ -6,14 +6,22 @@ class Article < ApplicationRecord
   has_many :articles_refunds, dependent: :restrict_with_error
   has_many :refunds, through: :articles_refunds
 
+  monetize :price_cents, as: :price, allow_nil: false, numericality: { greater_than: 0 }
```

Comment:
It doesn't work with just this?
```suggestion
  monetize :price_cents, allow_nil: false, numericality: { greater_than: 0 }
```
By default it removes the suffix `_cents`